sklearn/feature_selection/_base.py

```diff
@@ -163,6 +164,7 @@ def get_feature_names_out(self, input_features=None):
         feature_names_out : ndarray of str objects
             Transformed feature names.
         """
+        check_is_fitted(self)
```

Comment:
I removed `n_features_in` from the previous commit because:
-For `SelectFromModel`, the `n_features_in` attribute is only defined if the underlying estimator exposes the attribute when fit.